terra-jupyter-base/Dockerfile

```diff
@@ -109,6 +109,14 @@ RUN pip3 -V \
  # For gcloud alpha storage support.
  && pip3 install google-crc32c --target /usr/lib/google-cloud-sdk/lib/third_party
 
+RUN git clone https://github.com/broadinstitute/cromshell.git \
+ && cd cromshell \
+ && git checkout cromshell_2.0 \
+ && pip install .
+
+# verify cromshell 2.0 is installed
+RUN cromshell-alpha version
```

Comment:
you can create a test notebook in this repo for verifying the image does install cromwell properly...similar to https://github.com/DataBiosphere/terra-docker/blob/master/terra-jupyter-python/tests/smoke_test.ipynb